docs/general/dns-providers.md

```diff
@@ -1125,3 +1125,19 @@ Blocks more ads and malware than AdGuard DNS thanks to more advanced syntax, but
 | DNS, IPv4      | Varies; see link above. |
 | DNS, IPv6      | Varies; see link above. |
 | DNSCrypt, IPv4 | Varies; see link above. |
+
+#### dnslow.me
+
+[dnslow.me](https://dnslow.me/) is an advertisement and threat blocking, privacy-first, encrypted DNS, also a free and open source project on GitHub.
+
+Only encrypted DNS protocols are provided, plain-text DNS is not supported here.
+
+All DNS requests will be protected with threat-intelligence feeds and randomly distributed to some other DNS resolvers for enhanced privacy.
+
+Privacy policy: Logging is only enabled to debug, and improve the service itself, minimize the false-positive blocking. All logs will only be existing for a very short time. No logs will be shared, sold, or exchanged with any 3rd-party.
```

Comment:
Maybe also be specific about what a short period of time means?

or

`Privacy policy: logging is only used to debug, improve the service, and minimize false-positive blocking. All logs are stored for a short period of time only. Logs aren't shared, sold, or exchanged with any third party.`